types/it-pushable/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for it-pushable 1.3
+// Project: https://github.com/alanshaw/it-pushable
+// Definitions by: Junxiao Shi <https://github.com/yoursunny>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.2
+
+/// <reference types="node" />
```

Comment:
I’m probably overlooking something obvious, but why is this needed?

Reply:
It's needed for `AsyncIterable`. I'm going to change it to a "lib" reference.